src/sections/ServicesHero/index.css

```diff
@@ -5,7 +5,7 @@ $image-tablet-height: 340px;
 $image-mobile-height: $spacing-112;
 
 .root {
-  margin-top: $spacing-28;
```

Comment:
Since you are removing margin you can remove root class from js file

Reply:
Yes, I forgot to remove it during experiments.